src/utils/currencies.js

```diff
@@ -12,9 +12,9 @@ const CurrencySymbol = {
     symbol: '€',
     code: 'EUR'
   },
-  CNY: {
+  RMB: {
     symbol: '¥',
-    code: 'CNY'
+    code: 'RMB'
```

Comment:
Is this the code supported by the api we use?

Reply:
`CNY` is the proper one

Reply:
So international and api supported is `CNY` I'm waiting for the confirmation on what the display value should be. But anyway I've reverted this to `CNY`.